modules/virt-troubleshooting-cert-rotation-parameters.adoc

```diff
@@ -0,0 +1,33 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc
+
+[id="virt-troubleshooting-cert-rotation-parameters_{context}"]
+
+= Troubleshooting
+
+Deleting one or more `certConfig` values will cause them to revert to the default values, unless the default values conflict with the conditions noted above. If the default values conflict with those conditions, you will receive an error.
```

Comment:
s/will cause/causes

(This is another IBM style thing:
Tense
Write in the simple present tense as much as possible.
Use past or future tense only when you cannot use present tense or it does not make sense to use present tense.)

"the conditions noted above" --> a couple things. First, avoid using "above" as mentioned in an earlier comment. :) But the more pressing issue is that this is its own module, and it might be found outside of this assembly at some point. If you're going to refer to the conditions described in the YAML callouts, you need to also include those conditions in this module. It'll look repetitive, but to adhere to mod docs standards I'd suggest putting a list of the conditions somewhere in this module.